R/helper.R

```diff
@@ -0,0 +1,8 @@
+as_backend = function(id) {
```

Comment:
create a `zzz.R` file and doc the required imports, see https://github.com/mlr-org/mlr3data/blob/0b1f8fa1b6775cb39ac67d099550ea4420b3d0af/R/zzz.R#L1